encoder-otel-brave/pom.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
plase use spdx headers everywhere. the license plugin should catch this...

e.g. https://github.com/openzipkin/zipkin-gcp/blob/master/sender-pubsub/src/main/java/zipkin2/reporter/pubsub/PubSubSender.java#L1-L4

Reply:
Fixed in https://github.com/openzipkin-contrib/zipkin-otel/pull/7/commits/9f0dcdc630b2f8f1cbf422655a2fdcb54600e458